### ISSUE TITLE
Installation & Varnish Test Failed

### ISSUE THREAD

**Issue by purplecommerce** [magento/magento-cloud-docker#348]
### - Summary of the issue
Installation and varnish test failed while module on Marketplace.

### - Information on your environment,
This environment is failing:

"php_version":"7.4","magento_platform":"M2","magento_edition":"ce","magento_version":"2.3.7-p3","tool":"varnish","vendor":"purplecommerce","extension":"giftcard","version":"1.0.2","platform":"m2","magento_cloud_docker":"1.3.2","status":"FAIL"

"exit_code": 1

### - Steps to reproduce
If you check submission ID: c5x5xonyb1 you'll be able to cross-check the error.

**Comment by SamCritch** [comment]
@purplecommerce does it fail on the same as our extension?

`"command": "docker-compose -f docker-compose.yml run --rm build cloud-build"`

Our extension fails on that command with the following:

```
php ./vendor/bin/ece-tools schema:generate || true\n\nIn Processor.php line 129:\n                                                                               \n  The command \"php ./bin/magento module:enable --all --ansi --no-interaction\"  \n   failed. Warning: require(/app/setup/config/application.config.php): failed  \n   to open stream: No such file or directory in /app/vendor/magento/framework  \n  /Console/Cli.php on line 78                                                  \n                                                                               \n\nIn RefreshModules.php line 63:\n                                                                               \n  The command \"php ./bin/magento module:enable --all --ansi --no-interaction\"  \n   failed. Warning: require(/app/setup/config/application.config.php): failed  \n   to open stream: No such file or directory in /app/vendor/magento/framework  \n  /Console/Cli.php on line 78                                                  \n                                                                               \n\nIn Shell.php line 86:\n                                                                               \n  The command \"php ./bin/magento module:enable --all --ansi --no-interaction\"  \n   failed. Warning: require(/app/setup/config/application.config.php): failed  \n   to open stream: No such file or directory in /app/vendor/magento/framework  \n  /Console/Cli.php on line 78                                                  \n                                                                               \n\nrun <scenario>...\n\n
```

**Comment by purplecommerce** [comment]
@SamCritch yes you are right it fails on the same command with following details.

php .\/vendor\/bin\/ece-tools schema:generate || true\n\nIn Processor.php line 129:\n                                                                               \n  The command \"php .\/bin\/magento setup:di:compile --ansi --no-interaction\" fa  \n  iled. \u001b[32m%message%\u001b[39m 0\/9 [>---------------------------]   0% < 1 sec 6  \n  9.0 MiB\r\u001b[2K\u001b[32mProxies code generation...\u001b[39m 0\/9 [>--------------------  \n  ----]   0% < 1 sec 69.0 MiB\r\u001b[2K\u001b[32mProxies code generation...\u001b[39m 1\/9 [=  \n  =>----------------------]  11% < 1 sec 75.0 MiB\r\u001b[2K\u001b[32mRepositories code   \n  generation...\u001b[39m 1\/9 [==>-----------------]  11% < 1 sec 75.0 MiB          \n                                                                               \n\nIn CompileDi.php line 73:\n                                                                               \n  The command \"php .\/bin\/magento setup:di:compile --ansi --no-interaction\" fa  \n  iled. \u001b[32m%message%\u001b[39m 0\/9 [>---------------------------]   0% < 1 sec 6  \n  9.0 MiB\r\u001b[2K\u001b[32mProxies code generation...\u001b[39m 0\/9 [>--------------------  \n  ----]   0% < 1 sec 69.0 MiB\r\u001b[2K\u001b[32mProxies code generation...\u001b[39m 1\/9 [=  \n  =>----------------------]  11% < 1 sec 75.0 MiB\r\u001b[2K\u001b[32mRepositories code   \n  generation...\u001b[39m 1\/9 [==>-----------------]  11% < 1 sec 75.0 MiB          \n                                                                               \n\nIn Shell.php line 86:\n                                                                               \n  The command \"php .\/bin\/magento setup:di:compile --ansi --no-interaction\" fa  \n  iled. \u001b[32m%message%\u001b[39m 0\/9 [>---------------------------]   0% < 1 sec 6  \n  9.0 MiB\r\u001b[2K\u001b[32mProxies code generation...\u001b[39m 0\/9 [>--------------------  \n  ----]   0% < 1 sec 69.0 MiB\r\u001b[2K\u001b[32mProxies code generation...\u001b[39m 1\/9 [=  \n  =>----------------------]  11% < 1 sec 75.0 MiB\r\u001b[2K\u001b[32mRepositories code   \n  generation...\u001b[39m 1\/9 [==>-----------------]  11% < 1 sec 75.0 MiB          \n                                                                               \n\nrun <scenario>...\n\n
